### PR TITLE
Draft: Add Support for SplitPart

### PR DESCRIPTION
**Pull request by surya-palaniswamy**
Draft: Add Support for SplitPart

### CODE REVIEW

File: legend-engine-xt-relationalStore-sqlserver-pure/src/main/resources/core_relational_sqlserver/relational/tests/dbSpecificTests/customSqlServerTests.pure [hunk 1]
```diff
@@ -133,3 +133,18 @@ function <<test.Test>> meta::relational::tests::dbSpecificTests::sqlServer::test
    
    assertPlanGenerationForOptionalParameter(DatabaseType.SqlServer, $expectedPlan);
 }
+
```
Comment (vikask-gs) [inline legend-engine-xt-relationalStore-sqlserver-pure/src/main/resources/core_relational_sqlserver/relational/tests/dbSpecificTests/customSqlServerTests.pure:136]: we don't want to add more tests in this file, as we already db testing framework

File: legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/dbExtension.pure [hunk 2]
```diff
@@ -715,3 +715,16 @@ Class meta::relational::functions::sqlQueryToString::ToSql
    transform:meta::pure::metamodel::function::Function<{String[*]->String[*]}>[0..1];
    parametersWithinWhenClause : Boolean[*];
 }
+
+
+function meta::relational::functions::sqlQueryToString::processSplitPart():LambdaFunction<{String[3]->String[1]}>[1]
```
Comment (vikask-gs) [inline legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/dbExtension.pure:720]: move this fn to defaultExtension.pure . dbExtension.pure is only for api, for shared fns across <dbType>Extension.pure, use defaultExtension.pure

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**